awx/ui_next/src/screens/Job/Job.jsx

```diff
@@ -124,12 +126,24 @@ class Job extends Component {
               <Route
                 key="details"
                 path="/jobs/:type/:id/details"
-                render={() => <JobDetail type={match.params.type} job={job} />}
+                render={() =>
+                  match.params.type === 'workflow' ? (
+                    <WorkflowDetail job={job} />
+                  ) : (
+                    <JobDetail type={match.params.type} job={job} />
+                  )
+                }
               />,
               <Route
                 key="output"
                 path="/jobs/:type/:id/output"
-                render={() => <JobOutput type={match.params.type} job={job} />}
+                render={() =>
+                  match.params.type === 'workflow' ? (
+                    <WorkflowOutput job={job} />
```

Comment:
The job getting passed down to WorkflowOutput is stale bc we are trying fetch a workflow with the wrong id. 
![wf](https://user-images.githubusercontent.com/15881645/73373105-388c6f80-4286-11ea-98b9-58eaff6f51d4.gif)